setup.py

```diff
@@ -0,0 +1,32 @@
+from setuptools import setup, find_packages
+
+setup(
+    name='pyglidein',
+    use_scm_version=True,
+    setup_requires=['setuptools_scm'],
+    description='Some python scripts to launch HTCondor glideins',
+    url='https://github.com/WIPACrepo/pyglidein',
+    author='WIPAC',
+    author_email='contact-us@icecube.wisc.edu',
+    classifiers=[
+        'Programming Language :: Python :: 2',
+        'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.3',
+        'Programming Language :: Python :: 3.4',
+        'Programming Language :: Python :: 3.5',
+    ],
```

Comment:
What about python 3.6? :)

Reply:
And what about 2.6? Unless we assume we run parrot, cvmfs, or modules everywhere

Reply:
@briedel noticed that 2.6 isn't listed.  We probably need to continue support for that until we stop running on SL6.

Reply:
@briedel @dsschult I confirmed that Minio completely fails on 2.6.  I'm going to so something like:

`'minio;python_version>"2.7"',` in the setup.py file
and add conditional imports of minio in the client_util module.  

For now log shipping on python 2.6 will just have to be disabled.